src/components/circular-progress/circular-progress.tsx

```diff
@@ -0,0 +1,82 @@
+import { Component, h, Prop } from '@stencil/core';
+import { CircularProgressSize } from './circular-progress.types';
+// import { CircularProgressSize } from '@limetech/lime-elements'; // Kia: I'd like something like this, but it won't work :/
```

Comment:
yes sir

Reply:
It's not an export, it's an import. It's the same import as the line above, but I think Kia means that he would like to use the alias `'@limetech/lime-elements'` instead of the path `'./circular-progress.types'`.

The alias way for importing works sometimes, but when the _Types_ section of the docs broke recently, it was because a type was being imported using `from '@limetech/lime-elements'`, so I think it might be best to avoid it, at least for types.

Reply:
I know what he meant. If you want to use like this `'@limetech/lime-elements'`, you need to export `circular-progress.types` file `export * from './components/circular-progress/circular-progress.types';` in `interface.d.ts` file. That makes it available for the customer to use the types as well. I was just explaining how and when we use it.

Reply:
Ah! Now I understand what you meant! 👍